src/tools/schema2packager.ts

```diff
@@ -7,25 +7,36 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-
 import fs from 'fs';
 import path from 'path';
 import minimist from 'minimist';
-
 import {Manifest} from '../runtime/manifest.js';
-
-// TODO: options: output dir, filter specific schema(s)
-const argv = minimist(process.argv.slice(2), {
-  boolean: ['help'],
+import {Schema} from '../runtime/schema.js';
+import {Dictionary} from '../runtime/hot.js';
+
+const opts = minimist(process.argv.slice(2), {
+  string: ['outdir'],
```

Comment:
These changes will have to go into the new schema2base, but that seems easy enough

Reply:
Yep, will do.